src/components/AccessibilityDocumentsList/index.tsx

```diff
@@ -52,7 +72,7 @@ const AccessibilityDocumentsList = ({
               <Link target="_blank" rel="noreferrer" href={row.original.url}>
                 {t('documentTable.view')}
                 <span className="usa-sr-only">
-                  document type {/* TODO replace with real doc type */}
+                  document type {row.original.documentType}
```

Comment:
You can replace the entire "document type" string here, no need to keep that bit. So, the screen reader should read "View testing vpat" or "View special document type string". And I think you want `getDocType(row.original.documentType)`?

Reply:
I think you should be able to do. Please let me know if this doesn't work.
```
<Link target="_blank" rel="noreferrer" href={row.original.url} aria-label={`View ${getDocType(row.original.documentType)} in a new tab or window`}>
  {t('documentTable.view')}
</Link>
```